### PR TITLE
6.2 Release Notes Cover

### PR DESCRIPTION
**Pull request by sf1919**
**Description of work.**
The cover for the v6.2 Release Notes.

**To test:**

- remove build/docs/doctrees
- build the documents again and check there are no warnings
- Check the information is correct
- Are the images visible and clearly illustrate the new feature/improvement?
-  Check that the links work
-  Grammar/Spelling check

*There is no associated issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/release/v6.2.0/index.rst [hunk 1]
```diff
@@ -4,9 +4,9 @@
 Mantid 6.2.0 Release Notes
 ===========================
 
-.. figure:: ../../images/ImageNotFound.png
+.. figure:: ../../images/ReleaseNotesIndex_v6_2.png
```
Comment (martyngigg) [inline docs/source/release/v6.2.0/index.rst:7]: This is a nice image. Were we not able to include the workspace calculator in the end?

File: docs/source/release/v6.2.0/index.rst [hunk 2]
```diff
@@ -16,7 +16,14 @@ Mantid 6.2.0 Release Notes
 
 We are proud to announce version 6.2.0 of Mantid.
 
-**TODO: Add paragraph summarizing big changes**
+This release sees many new algorithms added or improved to extend the functionality of Mantid for users. In addition to this there are some significant new features
+including:-
+
+- Superplot, a new detector widget for the plot window that facilitates over-plotting and the manipulation of overplotted data.
```
Comment (martyngigg) [inline docs/source/release/v6.2.0/index.rst:22]: Can I ask what we mean by detector widget here? The spectra are related to detectors but they don't have to be...

**Review by martyngigg — CHANGES_REQUESTED** [review]
This looks good. Just a couple of small questions.

Also, could we remove the "under construction" note now? I think we're far enough into the process that it's worth doing now.

File: docs/source/release/v6.2.0/index.rst (hunk 1 above)
Comment (gvardany) [reply docs/source/release/v6.2.0/index.rst:7]: Yes, I think it is worth to include, perhaps at full stretch in the bottom.

File: docs/source/release/v6.2.0/index.rst (hunk 2 above)
Comment (gvardany) [reply docs/source/release/v6.2.0/index.rst:22]: I think originally this was decorator, got mistaken during the clean-up. In any case I think it is a too technical term for the end user, so just saying a new widget is fine. I'd suggest:
`Superplot, a new widget that facilitates over-plotting and data comparison with large number of workspaces.`

Comment (sf1919) [reply docs/source/release/v6.2.0/index.rst]: Agreed - I misread the description from the workbench notes but I think @gvardany simplified sentence works much better.

File: docs/source/release/v6.2.0/index.rst (hunk 1 above)
Comment (sf1919) [reply docs/source/release/v6.2.0/index.rst:7]: I've added the workspace calculator image in but I haven't included it at full stretch as it distorts horribly.

Comment (gvardany) [reply docs/source/release/v6.2.0/index.rst:7]: Ahh, I meant to stretch the widget in workbench before taking the screenshot, but taking the existing screenshot is fine as well.

**Review by martyngigg — CHANGES_REQUESTED** [review]
Thanks for making those changes. This is looking good.

I've just got one warning in the build output about a reference that it can't find:
```
﻿﻿/home/mgigg/Code/mantidproject/mantid-nightly/docs/source/release/v6.2.0/indirect_geometry.rst:56: WARNING: undefined label: algm-indirectillenertytransfer (if the link has no caption the label must precede a section header
```
Possibly just a typo with `enerty` -> `energy`?

**Comment by sf1919** [comment]
> 
> 
> Thanks for making those changes. This is looking good.
> 
> I've just got one warning in the build output about a reference that it can't find:
> 
> ```
> ﻿﻿/home/mgigg/Code/mantidproject/mantid-nightly/docs/source/release/v6.2.0/indirect_geometry.rst:56: WARNING: undefined label: algm-indirectillenertytransfer (if the link has no caption the label must precede a section header
> ```
> 
> Possibly just a typo with `enerty` -> `energy`?

To avoid a merge conflict I think this should be updated in #32559 rather than this PR.

**Review by martyngigg — APPROVED** [review]
Yep that makes perfect sense.

**Pull request merged**